Minor Correction (#1585)
substrate-developer-hub/substrate-docs: content/md/en/docs/tutorials/get-started/authorize-specific-nodes.md

2. Select **nodeAuthorization** and select **claimNode(node)**.

> 3. Copy and paste the hex-encoded peer identifier for the node owned by Dave(Do not forget to add prefix 0x), then click **Submit Transaction**.

Reviewer: ```suggestion
3. Copy and paste the hex-encoded peer identifier for the node owned by Dave after the required 0x prefix, then click **Submit Transaction**.
```
